### PR TITLE
fix: Not being able to send to stdenv stdin (for the scheduler) 

### PR DESCRIPTION
**Pull request by disconsented**

### CODE REVIEW

File: operations/message/console.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+ Copyright 2016 Padduck, LLC
```
Comment (LordRalex) [inline operations/message/console.go:2]: Update copyright to current year, as this file is new

File: operations/console/factory.go [hunk 2]
```diff
@@ -26,8 +26,8 @@ type OperationFactory struct {
 }
 
 func (of OperationFactory) Create(op pufferpanel.CreateOperation) (pufferpanel.Operation, error) {
-	message := cast.ToString(op.OperationArgs["message"])
-	return &Console{Text: message}, nil
+	input := cast.ToString(op.OperationArgs["input"])
```
Comment (LordRalex) [inline operations/console/factory.go:29]: Technically, this is a breaking change.

**Comment by LordRalex** [comment]
Please rebase and target v3, as that's the current dev pipeline.

**Comment by disconsented** [comment]
(this has been since superseded by other V3 work)

**Pull request closed without merging**